Packs/FeedZoom/Integrations/FeedZoom/README.md

```diff
@@ -6,6 +6,10 @@ One example of these providers is Zoom.
 
 This pack aims to address this issue by automating the collection of endpoint data in the form of an indicator feed. This will facilitate validation of the indicators before using them in enforcement points, for example firewalls, proxies, and more.
 
+## Note regarding Fetch Interval Time Parameter
+
+The default value of this parameter is 240 minutes, setting more frequent fetch interval may cause errors from the vendor.
```

Comment:
better to have this in the additionalinfo field of the parameter.
In addition, let's regenerate the readme to be in our conventions (with the parameters, command info, etc) and add the existing content to it (and then the additionalinfo above will appear here too).